extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -127,6 +127,24 @@ Results can feed its state from quizzes if all categories also specify `option-{
 </amp-story-interactive-results>
 [/sourcecode]
 
+### amp-story-interactive-slider
+
+The `amp-story-interactive-slider` element provides a voting experience based on a range. The options take the form of values along a horizontal line, where all options are valid. When selected, an average indicator and the average response are displayed.
```

Comment:
In this context, I think "where all options are valid" is intrinsic to the type (sliders usually don't have right/wrong answers), therefore not needed. We should explain that this component interacts by sliding the thumb though (as opposed to all the other components which are tappable).